packages/core/test/application_init_spec.ts

```diff
@@ -66,6 +67,51 @@ import {inject, TestBed, waitForAsync} from '../testing';
              resolve(null);
            });
 
+           expect(status.done).toBe(false);
+           status.donePromise.then(() => {
+             expect(status.done).toBe(true);
+             expect(completerResolver).toBe(true);
+           });
+         })));
+    });
+    describe('with async observable initializers', () => {
```

Comment:
Small suggestion to update block title (and also adding an extra line):
```suggestion
    });

    describe('with app initializers represented using observables', () => {
```